### DISCUSSION TITLE
How to get 2FA to work (Windows)

### DISCUSSION THREAD

**Discussion by davfriGH** [PiotrMachowski/Xiaomi-cloud-tokens-extractor#99] [Q&A]
Hi. I'm trying to use the token_extractor.exe, but I cannot get the 2FA to work. I open the 2FA link in my web browser and I successfully obtain and enter the verification code from Xiaomi, but once I try to log in again with token_extractor.exe, I get the same message saying 2FA is required.

Can you help me?

**Comment by PiotrMachowski** [comment]
Check out this discussion https://github.com/PiotrMachowski/Xiaomi-cloud-tokens-extractor/issues/43
